.github/workflows/docsbuild-release.yaml

```diff
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: 'Release Version. Example: 11.1.0'
+        required: true
+      ansible-major-version:
+        description: 'Example 11'
+        required: true
+
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible-version }}" porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CI_PR_BODY: |
+
+      ##### SUMMARY
+
+      ##### ISSUE TYPE
+
+      - Docs Pull Request
+
+      ##### COMPONENT NAME
+
+      docs/docsite/rst/porting_guides/'porting_guide_"${{ inputs.ansible-major-version }}".rst'
+
+jobs:
+  build:
+    name: 'Build PortingGuide (${{ inputs.ansible-version }})'
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
```suggestion
    steps:

    - name: Retrieve the major version component
      id: compute-version
      run: |
        echo -n 'major-version=' >> "${GITHUB_OUTPUT}"
        echo '${{ inputs.ansible-version }}' | sed 's#\([0-9]\+\)\..*#\1#' >> "${GITHUB_OUTPUT}"

```

Reply:
@anweshadas looks like you copy-pasted this but didn't mark as resolved. Marking as resolved now.

P.S. In the future, I recommend accepting from the GH UI since it marks as resolved automatically and correctly, plus credits the contribution properly.